style.css

```diff
@@ -184,3 +217,51 @@
 .step-img {
   width: 35%;
 }
+
+@media (max-width: 1200px) {
```

Comment:
Each @media needs to go below the appropriate class instead of long list of media queries.

So if you have .hero-img-mobile then add the media query for this class below so each class styles are grouped together